viewer/v2client/src/components/Inspector.vue

```diff
@@ -151,6 +152,7 @@ export default {
       const fetchUrl = `${this.settings.apiPath}/${this.documentId}/data.jsonld?${randomHash}`;
       fetch(fetchUrl).then((response) => {
         if (response.status === 200) {
+          this.documentETag = response.headers.get('ETag').replace(/"/g, ''); // Fix double quoted string
```

Comment:
Why strip quoting?

Reply:
Backend sends a quoted string in the ETag header. I would have assumed that frontend should treat it as an opaque value and not manupulate it in any way?

Reply:
Got it, will fix and update the PR.

Reply:
The keys is to make the subsequent `If-None-Match` identical. So if we get:
```
    ETag: "abc"
```
Do we send:
```
   If-None-Match: "abc"
```
?